### PR TITLE
Moved sheet Ids and updated other files

### LINKED ISSUE #25: Move sheet Ids
Sheet Id's currently exist in the 'checkins.ts'. They need moved to their own util file and the imports that use them need to be updated.

### PR DESCRIPTION
**Pull request by SebastianNeiswanger**
closes #25

### CODE REVIEW

File: src/utils/sheetID.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const EVENT_SHEET_ID = "EVENTS";
```
Comment (SpBills) [inline src/utils/sheetID.ts]: I noticed that there are a bunch of files in the utils folder. First of all, our file naming scheme is snake_case. 

Next, don't have these be in their own file but instead put them in something applicable like `sheets.ts`

Comment (SpBills) [reply src/utils/sheetID.ts]: So, please rename all the files to their snake_case equivalent.

**Review by SpBills — CHANGES_REQUESTED** [review]

**Comment by SebastianNeiswanger** [comment]
Renamed util files and moved the ids to sheets.ts

**Review by SpBills — APPROVED** [review]
LGTM

**Pull request merged**